Content/Common/common.json

```diff
@@ -28,5 +28,7 @@
 	"common_unauthorized": "Unauthorized",
 	"common_rateLimitReached": "Gateway access blocked due to exceeding rate limit. Please wait a few minutes to retry.",
 	"common_badHttpResponseStatusCode": "Bad HTTP response status code %d",
-	"common_dismiss": "Dismiss"
+	"common_dismiss": "Dismiss",
+	"common_noMnemonicAlert_title": "Could Not Complete",
+	"common_noMnemonicAlert_text": "The required seed phrase is missing. Please return to the account and begin the recovery process."
```

Comment:
moving [these two](https://github.com/radixdlt/apps-localization-src/blob/main/Content/1.1%20-%20Transaction%20Review/transactionReview.json#L65-L66) here, as they weren't used in Transaction Review screen but in two unrelated flows.